src/app/features/permits/components/organisms/PermitDetails/PermitDetails.tsx

```diff
@@ -38,14 +38,14 @@ const PermitDetail: React.FC<Props> = ({ detail }) => {
                 <dt>Vergunning:</dt>
                 <dd>{permit_granted ? "ja" : "nee"}</dd>
                 { date_from && <dt>Begindatum:</dt> }
-                { date_from && <dd>{ date_from }</dd> }
-                { date_to && <dt>Einddatum:</dt> }
-                { date_to && <dd>{ date_to }</dd> }
+                { date_from && <dd>{ date_from || "-" }</dd> }
```

Comment:
I think I would prefer TypeScript's Nullish Coalescing instead of `&&` and `||` in those situations

Reply:
Good point, fixed it